app/controllers/users_controller.rb

```diff
@@ -23,8 +26,11 @@ def show
 
   private
 
-    def user_params
-      params.require(:user).permit :full_name, :email, :password
-    end
+  def user_params
+    params.require(:user).permit :full_name, :email, :password, :invite_token
```

Comment:
`invite_token` does not really have to be an attribute on `User`. its fine if you'd use `hidden_field_tag :invite_token`.